docs/api/inputs/chat.md

```diff
@@ -207,6 +207,13 @@ mo.ui.chat(
 )
 ```
 
+```{eval-rst}
+.. autoclass:: marimo.ai.llm.google
+  :members:
+
+  .. autoclasstoc:: marimo._plugins.ui._impl.chat.llm.google
+```
```

Comment:
thanks for adding this one, can we add the groq one too?

Reply:
Added it now.